lusSTR/scripts/marker.py

```diff
@@ -376,7 +381,13 @@ def convert(self):
             bracketed_form = collapse_repeats_by_length(self.uas_sequence, 4)
         else:
             for m in re.finditer("GGGCTGCCTA", self.uas_sequence):
+                print(m)
                 break_point = m.end()
+            try:
+                break_point
+            except NameError:
+                for m in re.finditer("TTTT", self.uas_sequence):
+                    break_point = m.end() + 10
```

Comment:
While running some STRait Razor data through lusSTR, I ran into an error that it couldn't find ```GGGCTGCCTA``` within the sequence (there was a SNP within it) and therefore couldn't identify the break point. I'm surprised I haven't run into this error before. This identifies a run of Ts before that sequence. This allowed the entire STRait Razor file to run.